libs/dataset-explorer/src/lib/VISIONDatasetExplorerTab.styles.ts

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  IStyle,
+  mergeStyleSets,
+  IProcessedStyleSet,
+  getTheme
+} from "@fluentui/react";
+import { descriptionMaxWidth, FabricStyles } from "@responsible-ai/core-ui";
+
+export interface IDatasetExplorerTabStyles {
+  boldText: IStyle;
+  callout: IStyle;
+  chartContainer: IStyle;
+  chartEditorButton: IStyle;
+  chartWithAxes: IStyle;
+  chartWithVertical: IStyle;
+  cohortDropdown: IStyle;
+  cohortPickerInnerWrapper: IStyle;
+  cohortPickerWrapper: IStyle;
+  cohortPickerLabel: IStyle;
+  cohortPickerLabelWrapper: IStyle;
+  createCohortButton: IStyle;
+  createCohortButtonActive: IStyle;
+  createCohortText: IStyle;
+  colorBox: IStyle;
+  colorValue: IStyle;
+  verticalAxis: IStyle;
+  rotatedVerticalBox: IStyle;
+  legendAndText: IStyle;
+  horizontalAxisWithPadding: IStyle;
+  paddingDiv: IStyle;
+  horizontalAxis: IStyle;
+  page: IStyle;
+  infoIcon: IStyle;
+  helperText: IStyle;
+  infoWithText: IStyle;
+  individualChartContainer: IStyle;
+  mainArea: IStyle;
+  legendLabel: IStyle;
+  legendItem: IStyle;
+  searchAndFilterContainer: IStyle;
+  searchBox: IStyle;
+  filter: IStyle;
+  filterIcon: IStyle;
+  filterPlaceholderText: IStyle;
+  toolBarContainer: IStyle;
+  mainContainer: IStyle;
+  titleBar: IStyle;
+  titleBarLabel: IStyle;
+  titleBarNumber: IStyle;
+  errorIcon: IStyle;
+  successIcon: IStyle;
+  imagesContainer: IStyle;
+  slider: IStyle;
+  smallItalic: IStyle;
+  sidePanel: IStyle;
+  tabs: IStyle;
+}
+
+export const datasetExplorerTabStyles: () => IProcessedStyleSet<IDatasetExplorerTabStyles> =
+  () => {
+    const theme = getTheme();
+    return mergeStyleSets<IDatasetExplorerTabStyles>({
+      boldText: {
+        fontWeight: "600",
```

Comment:
we are tyring to optimize styles.
Does office fabric component <Label> work here?